apps/codeforafrica/src/theme/index.js

```diff
@@ -0,0 +1,282 @@
+import { createTheme } from "@commons-ui/core";
+
+const FONT_FAMILY_PRIMARY = "'Open Sans', sans-serif";
+const FONT_FAMILY_SECONDARY = "'Merriweather', serif";
+
+const palette = {
+  mode: "light",
+  primary: { main: "#1020E1", light: "#EFF0FD", dark: "#08125C" },
+  secondary: { main: "#000000", light: "#7F7272" },
+  text: {
+    primary: { main: "#000000" },
+    secondary: { main: "#FFFFFF" },
+  },
+  background: { main: "#F6F5F5" },
+  highlight: { main: "#ED1C24" },
+  grey: { main: "#B4ABAB", light: "#E3DFDF" },
+};
+
+const breakpoints = {
+  values: {
+    xs: 390,
+    sm: 768,
+    md: 1152,
+    lg: 1440,
+    xl: 1920,
+  },
+};
+
+const theme = createTheme({
+  palette,
+  breakpoints,
+  typography: {
+    FONT_FAMILY_PRIMARY,
+    FONT_FAMILY_SECONDARY,
+    display1: {
+      fontFamily: FONT_FAMILY_PRIMARY,
+      fontStyle: "normal",
+      fontWeight: 300,
+    },
+    display2: {
+      fontFamily: FONT_FAMILY_PRIMARY,
+      fontStyle: "normal",
+      fontWeight: 300,
+    },
+    h1: {
+      fontFamily: FONT_FAMILY_PRIMARY,
+      fontStyle: "normal",
+      fontWeight: 300,
+      letterSpacing: "-0.04em",
+    },
+    h2: {
+      fontFamily: FONT_FAMILY_PRIMARY,
+      fontStyle: "normal",
+      fontWeight: 300,
+      letterSpacing: "-0.02em",
+    },
+    h3: {
+      fontFamily: FONT_FAMILY_PRIMARY,
+      fontStyle: "normal",
+      fontWeight: 300,
+    },
+    h4: {
+      fontFamily: FONT_FAMILY_PRIMARY,
+      fontStyle: "normal",
+      fontWeight: 300,
+    },
+    h5: {
+      fontFamily: FONT_FAMILY_PRIMARY,
+      fontStyle: "normal",
+      fontWeight: 300,
+      letterSpacing: "-0.02em",
+    },
+    h6: {
+      fontFamily: FONT_FAMILY_PRIMARY,
+      fontStyle: "normal",
+      fontWeight: 300,
+      letterSpacing: "-0.02em",
+    },
+    subheading: {
+      fontFamily: FONT_FAMILY_SECONDARY,
+      fontStyle: "normal",
+      fontWeight: 300,
+    },
+    subtitle1: {
+      fontFamily: FONT_FAMILY_PRIMARY,
+      fontStyle: "normal",
+      fontWeight: 400,
+    },
+    subtitle2: {
+      fontFamily: FONT_FAMILY_PRIMARY,
+      fontStyle: "normal",
+      fontWeight: 400,
+    },
+    body1: {
+      fontFamily: FONT_FAMILY_PRIMARY,
+      fontStyle: "normal",
+      fontWeight: 400,
+    },
+    caption: {
+      fontFamily: FONT_FAMILY_PRIMARY,
+      fontStyle: "normal",
+      fontWeight: 400,
+    },
+  },
+  components: {
+    MuiButtonBase: {
+      defaultProps: {
+        disableRipple: true,
+      },
+    },
+    MuiButton: {
+      variants: [
+        {
+          props: { variant: "contained", color: "primary" },
+          style: {
+            backgroundColor: palette.primary.main, // #1020E1 => light blue
```

Comment:
Move the comment `// #1020E1 => light blue` to palette definition. Otherwise, it's repeated everywhere.